src/routes/DigitalAssets/routes/Send/index.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+
+import DigitalAssetsSendViewContainer from './DigitalAssetsSendViewContainer'
+
+const send = {
+  path: 'send',
+  component: DigitalAssetsSendViewContainer,
+}
+
+const sendAsset = {
+  path: 'send/:asset/',
+  component: DigitalAssetsSendViewContainer,
+}
+
+const sendAssetTo = {
+  path: 'send/:asset/:to',
+  component: DigitalAssetsSendViewContainer,
+}
+
+const repeatPayment = {
+  path: 'send/repaat/:txhash',
+  component: DigitalAssetsSendViewContainer,
+}
```

Comment:
@stcherenkov 
it should be just pre-filled form, by clicking on `repeat` button.
we really must not over complicate such simple features.
especially when we already have ready implementation.